### DISCUSSION TITLE
Extra plotting parameters possible?

### DISCUSSION THREAD

**Discussion by whattheflux** [guydavis/machinaris#871] [Q&A]
Is it possible to add extra parameters to plotting for gigahorse?
I have a pretty low end GPU that does work if I just run it from an unraid console, but I need to set streams to 2 to run inside machinaris.
I can't see a way to add streams. When i add streams:2 or -S: 2 to the plotman config it gives an error.

**Comment by guydavis** [comment]
Hi, sorry for the confusion.  Please try this in the Settings | Plotting page:

```
gpu_streams: 2
```

I'll go improve the [documentation](https://github.com/guydavis/machinaris/wiki/Gigahorse#config) on this now.
